Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG PROJ_VERSION="6.3.0"
+
+FROM rust:1-slim-stretch as builder
+ARG PROJ_VERSION
+ENV PROJ_DEB "proj_${PROJ_VERSION}_amd64.deb"
+ENV GPG_KEY "C60D758F807A525534C5DFD57B639E3638A8112A"
+WORKDIR /usr/src/app
```

Comment:
How about moving this line below, just above the `COPY` operation since they're related.